bundles/org.openhab.binding.sony/src/main/resources/OSGI-INF/SonyDefinitionProviderImpl.properties

```diff
@@ -0,0 +1,27 @@
+SonyFolderSource.Folder.ThingTypes = /db/local/types
```

Comment:
yep - there was no way around that.  the api key in question is very limited in what it can do (basically post issues) - was the only way to use github like that.